pkg/meta/lib/future_extensions.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Annotations that developers can use to extend [Future] capabilities.
```

Comment:
This is wrong in a couple of ways. First, it isn't defining annotations, it's defining extension methods. Second, the things it's declaring can't be used to extend the capabilities of `Future`, they can be used to control the behavior of a lint rule.